sonic-xcvrd/scripts/xcvrd

```diff
@@ -1236,6 +1238,10 @@ class DaemonXcvrd(daemon_base.DaemonBase):
         self.log_info("Init port sfp status table")
         init_port_sfp_status_tbl(self.stop_event)
 
+        # Init port y_cable status table
+        self.log_info("Init port sfp status table")
```

Comment:
Did you intend to update this log message?

Reply:
I have removed the log message for now, but if the Y cable presence is established, should we then log a message ?

Reply:
If you think it will be helpful for debugging, sure. But this message was incorrect, and it was logged unconditionally here, not only if Y-cable presence was detected. It should probably be done in init_ports_status_for_y_cable().